src/TraceEvent/ETWTraceEventSource.cs

```diff
@@ -900,7 +900,8 @@ private bool TraceEventBufferCallback(IntPtr rawLogFile)
         // Used to give TraceLogging events Event IDs. 
         private TraceLoggingEventId traceLoggingEventId;
 
-        internal override string ProcessName(int processID, long time100ns)
+        [Obsolete("This is experimental, you should not use it yet for non-experimental purposes.")]
+        public override string ProcessName(int processID, long time100ns)
```

Comment:
I think it is fine to make these public without the [Obsolete] attribute.

Reply:
I will make these public then. Without the attribute.

---
In reply to: [385882429](https://github.com/microsoft/perfview/pull/1107#discussion_r385882429) [](ancestors = 385882429)